executor/lockstats/lock_stats.go

```diff
@@ -24,28 +24,17 @@ import (
 	"github.com/pingcap/tidb/util/chunk"
 )
 
-var _ exec.Executor = &LockStatsExec{}
+var _ exec.Executor = &Exec{}
 var _ exec.Executor = &UnlockStatsExec{}
```

Comment:
Looks like it's better to make their names more consistent.

Reply:
This is suggested by the liner. In golang, we prefer the `lockstats.Exec` instead of `lockstats.LockStatsExec.`

Reply:
I see 😂 Let's merge it anyway.

Reply:
I renamed them to `LockExec` and `UnlockExec`.